documentation/RepoOrganization.md

```diff
@@ -16,6 +16,7 @@ Some notes on the directory layout organization in this repo.
 
     - [Mlos Settings System Code Generation System](../source/Mlos.SettingsSystem.CodeGen/)
     - [Mlos.Core Shared Memory Communication Channel](../source/Mlos.Core/doc/)
+    - [Mlos.Agent.Server](../source/Mlos.Agent.Server/)

```

Comment:
Should it also be in the website menu?

Reply:
Should we add a "Components Documentation" section and list this under that?

Reply:
642b39d1acc9e07b62e37f67ca32f0c3905b0c37 adds this